src/scripts/apiGateway-js-sdk/lib/axios/dist/axios.standalone.js

```diff
@@ -1,5 +1,5 @@
 /*
-axios v0.7.0 
+axios v0.7.0
```

Comment:
remove this file from the branch. to do that switch to the lazyLoading branch and run:

1) git checkout master src/scripts/apiGateway-js-sdk/lib/axios/dist/axios.standalone.js
2) git add -A
3) git commit -m "<commit message>"